harvester_e2e_tests/integration/test_hosts.py

```diff
@@ -67,3 +127,156 @@ def test_host_poweron_state(api_client, host_state, wait_timeout):
 
     _, node = api_client.hosts.get(node['id'])
     assert "active" == node["metadata"]["state"]["name"]
+
+
+@pytest.mark.hosts
+@pytest.mark.p0
+def test_verify_host_info(api_client):
+    status_code, nodes_info = api_client.hosts.get()
+
+    assert status_code == 200, f"Failed to list nodes with error: {nodes_info}"
+    assert len(nodes_info['data']) >= 1, "Incorrect hosts amount"
+
+    for node in nodes_info.get('data', []):
+        node_name = node.get('metadata', {}).get('name', "")
+        assert "" != node_name, ("Node name can't be empty")
+
+        cpus = node.get('status', {}).get('capacity', {}).get('cpu', 0)
+        assert 0 != cpus, (
+            f"Node {node_name} cpu should not be zero"
+        )
+
+        mems = node.get('status', {}).get('capacity', {}).get('memory', "0Ki")
+        mems = "".join(c for c in mems if c.isdigit())
+        mems = mems or "0"
+        assert 0 != int(mems, 10), (
```

Comment:
We need to check every nodes with every resources, so we can not assert in the for-loop directly.
In here, we should check them in another way, such like:
https://github.com/harvester/tests/blob/241d35b577e4a4d322a669a1c2f9c7e21ad57e22/harvester_e2e_tests/integration/test_vm_functions.py#L442-L468